portal-kernel/src/com/liferay/portal/kernel/transaction/Isolation.java

```diff
@@ -19,7 +19,7 @@
 import java.util.Map;
 
 /**
- * Defines how transations are visible to those outside the transaction.
+ * Defines the isolation level from other transactions.
```

Comment:
As an example, would you describe the READ_COMMITTED level?

Do you mean _this_ transaction is forbidden to do dirty reads?

Reply:
It's declarative. The transaction will not see any dirty data.

Reply:
You cannot try to do a dirty read, you can permit to see them if they are there.